express-app/models/estudio.js

```diff
@@ -48,13 +48,44 @@ class Estudio extends Document {
     this.status = {
       type: String,
       choices: opcionesStatusChoices,
-      default: 'Borrador',
+      default: opcionesStatusChoices[0],
```

Comment:
opcionesStatusChoices[0] is equal to "Aprobado", "Borrador" is the option 3 so this line would be opcionesStatusChoices[2]